README.md

```diff
@@ -20,36 +21,176 @@ The Commerce Layer checkout application (React) provides you with a PCI-complian
+
+#### Gift cards
+
+The Checkout application supports [gift cards](https://docs.commercelayer.io/developers/v/api-reference/gift_cards) and [coupon](https://docs.commercelayer.io/developers/v/api-reference/coupons) codes (just fill in the related field in the order summary). You can use a gift card or coupon to pay — in total or in part — for the order. A single gift card and a coupon code can also be used together to pay for the same order.
+
+#### Bundles
+
+The Checkout application supports [bundles](https://docs.commercelayer.io/developers/v/api-reference/bundles) and shows them as a single item both in the order summary and within the *Delivery* step. The SKUs belonging to bundles may still be split across more than one shipment, based on their availability and the selected inventory model strategy.
+
+#### Return to cart
+
+If the order has the attribute `cart_url` set, a "Return to cart" link will be displayed.
+
+### Customer step
+
+Here is where customers provide their email address, billing and shipping information.
+
+#### Logged customers
+
+If the access token used to build the checkout URL is a [customer token](https://docs.commercelayer.io/developers/authentication/password) customers can select one of their saved addresses from their address book and use it as the billing and/or shipping address. If the customer has only one address in the address book the Checkout application will use this address as the billing and shipping address and skip the *Customer* step.
+
+#### Digital products
+
+In the case of digital product purchases (i.e. SKUs with the `do_not_ship` flag enabled), the shipping address form doesn't appear.
+
+#### Locking the shipping country code
+
+If the order has the attribute `shipping_country_code_lock` set, customers can select only the specified country code in the shipping address form. If they select a different country for the billing address, the shipping address section will open automatically with the country code already selected and disabled.
+
+
+### Delivery step
+
+Here is where customers select a shipping method for each shipment of their order.
+
+#### Single shipping method
+
+If there is only one available [shipping method](https://docs.commercelayer.io/developers/v/api-reference/shipping_methods) per shipment, the Checkout application will autoselect that shipping method for each shipment of the order and skip the *Delivery* step.
+
+#### Shipping zone check
+
+If the shipping address is outside the associated [shipping zone](https://docs.commercelayer.io/developers/v/api-reference/shipping_zones) customers will receive an error and won't be able to place the order.
+
+#### Out-of-stock items
+
+If one of the items in the order is out of stock customers will get an error message and won't be able to place the order. If the order has the attribute `cart_url` set, the error message will contain a link to them go back and edit the shopping cart.
```

Comment:
Actually there was a word missing: it should have been "a link to _let_ them go back..." (meaning the customers). But it can work also as you suggest.

Reply:
yes, it works both ways